public/view/login.js

```diff
@@ -0,0 +1,35 @@
+import { loginGoogle, loginFB, loginUserEmail } from '../firebase.js';
+
+export default () => {
+    const root = document.getElementById('root')
+    const viewLogin = `
+    <div class="mainConteinerLogin">
+    <img src="img/portadaDesktop.jpg" alt="" class="imageLoginDesktop">
```

Comment:
traten de no dejar vacío los alt porque los buscadores usan esa etiqueta para ubicar las búsquedas, ademas favorece el SEO de una web 
```suggestion
    <img src="img/portadaDesktop.jpg" alt="algo" class="imageLoginDesktop">
```